tests/de/lock_HassTurnOn.yaml

```diff
@@ -0,0 +1,28 @@
+language: de
+tests:
+  - sentences:
+      - "schließe die Haustür zu"
+      - "Haustür zumachen"
+      - "Haustür absperren"
+    intent:
+      name: HassTurnOn
+      context:
+        domain: lock
+      slots:
+        name: Haustür
+    response: haustür abgeschlossen
+
+  - sentences:
```

Comment:
based on change above, we should add "mach die Haustür auf"

Reply:
this test sentence is still missing 🙂

Reply:
I've added it with an area :) but I can add it without an area, too

Reply:
ah ... my bad, i was in the wrong file (_we are in TurnOn/lock, not TornOff/unlock_) 🙈